app/views/proposal/collection_draft_proposals/progress.html.erb

```diff
@@ -3,7 +3,9 @@
   <section class="action wide-content">
     <div class="wide-content-inside">
       <span>
-        <span id="proposal-status-display-left" class="eui-badge eui-badge--split ">Proposal Status: <span><%= get_resource.proposal_status.titleize %></span></span>
+        <span class="eui-badge eui-badge--split proposal-status-badge"><%= get_resource.request_type == 'create' ? 'Draft Proposal Submission:' : "#{get_resource.request_type.titleize} Metadata Request:" %>
```

Comment:
I had suggested seeing if the text in the badge could be underlined, and if that made it look more like a link.